src/components/card/services/addToCart.service.js

```diff
@@ -12,11 +13,13 @@ const addToCartService = async ({ url, body, token, successStatus, cartDispatch,
                 payload: { cart: response.data.cart },
             });
             setCtaBtnText("Go To Cart");
+            toast.success(body.product.title.slice(0, 13).trim() + "... Added to Cart");
         } else {
             throw new Error(response.status, "<-- error code");
         }
     } catch (error) {
-        alert(error);
+        toast.error("Error Occured! Please Try Again.");
+        console.error(error)
```

Comment:
You can get error from the backend itself just write
`toast.error(error.response.data.errors[0])`

Reply:
Thanks for the suggestion.